retailcoder/Rubberduck
# Inno Setup Installer Script

Round 2 - Fight!

So I have taken on board the comments from the previous Pull Request and revised my submission.  I removed the existing installer projects and added a script to build a single installer using Inno Setup.

To test:
- You will need to build the Debug build as this is what the installer builds from.  The output path is hard-coded into the script but could easily be moved to point at a Release build if so desired.  Note that the post build events from the Debug build may interfere with installation, I'm not sure on this and would be keen for dialogue on this issue.
- You will also need to install Inno Setup and/or Inno Script Studio.  The latter is much better for working on scripts as there are a lot of GUIs to help.
- Once completing the above steps you should be able to compile the installer from Inno Script Studio.

I suggest that you pull my branch and test it thoroughly before committing to replacing the current installers.
